### PR TITLE
It is possible to have no active objectives

### PR DESCRIPTION
**Pull request by Bod9001**
better handling since it causes the admin to get disconnected if round restart is called before the objective is set

### CODE REVIEW

File: UnityProject/Assets/Scripts/Systems/StationObjectives/StationObjectiveManager.cs [hunk 1]
```diff
@@ -81,11 +81,18 @@ private string GetObjectiveStatus()
...
+			if (activeObjective != null)
+			{
+
+				bool complete = false;
+
+				complete = activeObjective.CheckCompletion();
+				message += $"{activeObjective.GetRoundEndReport()}";
+				message += complete ? "Completed\n" : "Failed\n";
+				return message;
+			}
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Systems/StationObjectives/StationObjectiveManager.cs:94]:
```suggestion
			var message = $"Status of {MatrixManager.MainStationMatrix.GameObject.scene.name}:\n";
			if (activeObjective != null)
			{
				message += $"{activeObjective.GetRoundEndReport()}";
				message += activeObjective.CheckCompletion() ? "Completed\n" : "Failed\n";
				return message;
			}
```

Codacy gonna complain about string builder too

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Systems/StationObjectives/StationObjectiveManager.cs:94]: Oh wait sorry you only added the null check, not the complete var stuff.

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Systems/StationObjectives/StationObjectiveManager.cs:94]: Resolve?

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Systems/StationObjectives/StationObjectiveManager.cs:94]: yeah

**Review by ThatDan123 — APPROVED** [review]

**Review by NoooneyDude — APPROVED** [review]

**Pull request merged**